### PR TITLE
Next small OC improvements

### PR DESCRIPTION
**Pull request by deg-pl**

### CODE REVIEW

File: lib/languages/nl.php [hunk 1]
```diff
@@ -201,7 +203,6 @@
     'new_password' => 'Nieuw wachtwoord',
     'new_topic' => 'New topic',
     'page_error' => 'Site error',
-    'contact' => 'Contact',
```
Comment (deg-pl) [inline lib/languages/nl.php]: @harrieklomp - there was a duplicate of 'contact' entry in language file. I've removed one, but one was "Contact us", the second one - only "Contact". Could you check it please - which is correct.

File: lib/languages/ro.php [hunk 2]
```diff
@@ -201,7 +203,6 @@
     'new_password' => 'Parolă nouă',
     'new_topic' => 'Subiect nou',
     'page_error' => 'Eroare site',
-    'contact' => 'Contact',
```
Comment (deg-pl) [inline lib/languages/ro.php]: @andrixnet there was a duplicate of 'contact' entry in language file. I've removed one, but one was "Contactează-ne", the second one - only "Contact". Could you check it please - which is correct.

File: lib/languages/nl.php (hunk 1 above)
Comment (harrieklomp) [inline lib/languages/nl.php]: I have corrected it to "Contact".

Comment (deg-pl) [inline lib/languages/nl.php]: Thanks!

**Pull request merged**